src/Arcus.Scripting.ApiManagement/Scripts/Upload-AzApiManagementSystemCertificate.ps1

```diff
@@ -0,0 +1,26 @@
+param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Resource group is required"),
+    [Parameter(Mandatory = $true)][string] $ServiceName = $(throw "API Management service name is required"),
+    [Parameter(Mandatory = $true)][string] $CertificateFilePath = $(throw "Certificate file-path is required"),
+    [Parameter(Mandatory = $false)][switch] $AsJob = $false
+)
+
+Write-Verbose "Loading public CA certificate '$CertificateFilePath'..."
+$rootCA = New-AzApiManagementSystemCertificate -StoreName "Root" -PfxPath $CertificateFilePath
+$systemCert = @($rootCa)
+Write-Host "Loaded public CA certificate '$CertificateFilePath'"
+
+Write-Verbose "Retrieving Azure API Management service '$ServiceName' instance..."
+$apimContext = Get-AzApiManagement -ResourceGroupName $ResourceGroupName -Name $ServiceName
```

Comment:
Can we include some kind of validation in case the APIM-instance cannot be found?
For instance if $apimContext -eq null -> Write-Error "Unable to find the Azure API Management Instance with name ... in resource group ..."
Perhaps something to take into account for the other scripts as well, as an improvement?

Reply:
Yeah, valid remark. Wasn't aware that this `Get-AzApiManagement` could return `$null`.

Reply:
The other scripts usually creates a new API Management context, is this susceptible to this too?

Reply:
This is indeed something that perhaps should be implemented on other scripts as well, just to have some additional line of validation to confirm whether the provided APIM-details are actually pointing towards a valid/existing instance.

Reply:
Created issue for this #220 . Resolving this one.